src/Extension/ExtendableTrait.php

```diff
@@ -122,6 +124,21 @@ public static function clearExtendedClasses()
         self::$extendableCallbacks = [];
     }
 
+    /**
+     * Normalizes the provided extension name
+     *
+     * @param string $name
+     * @return string
+     */
+    protected function extensionNormalizeClassName(string $name): string
+    {
+        $name = str_replace('.', '\\', trim($name));
+        if (!is_null($this->extensionGetClassLoader()) && ($alias = $this->extensionGetClassLoader()->getAlias($name))) {
```

Comment:
@bennothommo can we cache this locally as `$this->extensionClassLoader`? Every call to `extensionNormalizeClassName` now results in two calls to `class_exists()` and `App::make()`